### PR TITLE
fix typos for TestBackoffHighWaterMark

### PR DESCRIPTION
**Pull request by qjiang77**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: staging/src/k8s.io/client-go/util/flowcontrol/backoff_test.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ func TestBackoffReset(t *testing.T) {
 	}
 }
 
-func TestBackoffHightWaterMark(t *testing.T) {
+func TestBackoffHighWaterMark(t *testing.T) {
```
Comment (nikhita) [inline staging/src/k8s.io/client-go/util/flowcontrol/backoff_test.go]: @qjiang77 thanks for the clean up! :+1:

**Comment by qjiang77** [comment]
Hi, I have already signed the CLA. Could you please review it again? Thanks!
@nikhita @yliaog

**Comment by nikhita** [comment]
/assign @caesarxuchao 
for approval

**Review by idealhack — APPROVED** [review]

**Pull request merged**